config/sync/itk_pretix.pretixconfig.yml

```diff
@@ -0,0 +1,5 @@
+itk_pretix: null
+_core:
+  default_config_hash: peWh3JrwZ60PoKUGdO3_bHBdvmb6HQb6LscjOgivUjc
+langcode: da
+api_token: '1234567'
```

Comment:
We should not commit our api token. We should probably use tricks similar to the ones used in https://github.com/itk-dev/hoeringsportal/blob/master/config/sync/system_status.settings.yml.

Reply:
Changed config